src/main/java/org/kohsuke/github/GHOrganization.java

```diff
@@ -94,6 +94,17 @@ public GHTeam getTeamByName(String name) throws IOException {
     }
 
     /**
+     * Finds a team that has the given slug in its {@link GHTeam#getSlug()}
+     */
+    public GHTeam getTeamBySlug(String slug) throws IOException {
+        for (GHTeam t : listTeams()) {
+            if(t.getSlug().equals(slug))
```

Comment:
Irritating that the [single-team API](https://developer.github.com/v3/orgs/teams/#get-team) seems to require a numeric ID rather than the “slug”.